src/readiness.ts

```diff
@@ -29,7 +29,11 @@ async function tryUploadingSingleChunk(bee: Bee, stampsManager: StampsManager):
 
     return true
   } catch (error) {
-    logger.error('unable to upload chunk to bee', error)
+    if (error) {
+      // do not log the binary chunk data
+      Reflect.deleteProperty(error, 'requestOptions')
+    }
+    logger.error('unable to upload readiness-check chunk to bee', error)
```

Comment:
Hmm, what sort of structure the error has? I would maybe suggest instead of removing the `requestOptions`, to simply log only the error message like `error.message` if that would make sense.